Sources/Bot/Models/CommitState.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+public struct CommitState: Decodable, Equatable {
+
+    public enum State: String, Decodable {
+        case pending
+        case failure
+        case success
+    }
+
+    public struct Statuses: Equatable, Decodable {
```

Comment:
[nit] Feels strange that the type is named in plural while it only represents a single instance.
It's only plural because you use it in an array down below, but an array of strings is `[String]` not `[Strings]`, so an array of Statuses should be `let statuses: [Status]` not `let statuses: [Statuses]` :wink:

Reply:
That's what you get when you copy-paste something from the API without actually processing it 🤦‍♂